lib/deploy/events/schedule/compileScheduledEvents.js

```diff
@@ -95,11 +95,17 @@ module.exports = {
                     ${InputPath ? `"InputPath": "${InputPath}",` : ''}
                     "Arn": { "Ref": "${stateMachineLogicalId}" },
                     "Id": "${scheduleId}",
-                    "RoleArn": {
-                      "Fn::GetAtt": [
-                        "${scheduleIamRoleLogicalId}",
-                        "Arn"
-                      ]
+                    "RoleArn": ${
```

Comment:
instead of inlining everything, how apart pulling it out, e.g. `const roleArn = ...`? not essential, just makes it a bit easier to see the template

Reply:
Agreed, done.